### ISSUE TITLE
twitter-text-1.5.0 permission is 640

### ISSUE THREAD

**Issue by hsbt** [twitter-archive/twitter-text-rb#81]
twitter-text-1.5.0 is not allowed to read for other owner. it's following situation.

```
% gem install twitter-text
% gem unpack twitter-text
% cd twitter-text-1.5.0/lib
% ls -l
drwxr-xr-x 10 hsbt wheel 340  1 12 22:01 twitter-text
-rw-r-----  1 hsbt wheel 564  1 12 22:01 twitter-text.rb
% ls -l twitter-text
-rw-r----- 1 hsbt wheel 22215  1 12 22:01 autolink.rb
-rw-r----- 1 hsbt wheel   447  1 12 22:01 deprecation.rb
-rw-r----- 1 hsbt wheel 12488  1 12 22:01 extractor.rb
-rw-r----- 1 hsbt wheel  2784  1 12 22:01 hit_highlighter.rb
-rw-r----- 1 hsbt wheel 17123  1 12 22:01 regex.rb
-rw-r----- 1 hsbt wheel  1941  1 12 22:01 rewriter.rb
-rw-r----- 1 hsbt wheel   916  1 12 22:01 unicode.rb
-rw-r----- 1 hsbt wheel  4614  1 12 22:01 validation.rb
```

For example, application user installed twitter-text, but apache2 user used twitter-text. it happend LoadError.

Please adding to other users reading permission and bump version 1.5.1.

Thanks.

**Comment by jlxw** [comment]
Yes the gem is only readable by the owner of the file so if the ruby process is not root or user that installed the gem, the gem will not load. This needs to be fixed by the person pushing the gem to rubygems, permission needs to be changes to world-readable and gem re-pushed.

The workaround I am using is to use the github version of the gem in my Gemfile: 

gem 'twitter-text', github: 'twitter/twitter-text-rb'

**Comment by hsbt** [comment]
I confirmed to solved this issue on twitter-text-1.6.0, Thank you so much :octocat:
